infrastructure/zk/src/fmt.ts

```diff
@@ -61,6 +62,13 @@ export const command = new Command('fmt')
                 case 'system-contracts':
                     await prettierSystemContracts(cmd.check);
                     break;
+                case 'sqlx-queries':
+                    if (cmd.check) {
+                        console.warn('format-sqlx-queries does not support --check, skipping!');
```

Comment:
Wouldn't it be possible to support checking by parsing all `sqlx` queries and checking for each of them that the formatted version is exactly equal to the original version?

Reply:
corrected